binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/HueThingHandlerFactory.java

```diff
@@ -116,4 +118,12 @@ protected synchronized void removeHandler(ThingHandler thingHandler) {
             }
         }
     }
+
+    @Override
+    protected Map<DefaultPropertyKey, String> getStaticProperties(Thing thing) {
+        Map<DefaultPropertyKey, String> staticProperties = super.getStaticProperties(thing);
+        staticProperties.put(DefaultPropertyKey.VENDOR, "Philips");
```

Comment:
This is not correct - the hue binding also supports bulbs from Osram and GE and potentially any other bulb that registers as a "Generic_ZLL" type.